build/repo.targets

```diff
@@ -1,9 +1,5 @@
 <Project>
-  <Target Name="CleanNpm" AfterTargets="Clean">
-    <RemoveDir Directories="$(RepositoryRoot)node_modules" />
-  </Target>
-
-  <Target Name="RestoreNpm" AfterTargets="Restore">
+  <Target Name="RestoreNpm">
```

Comment:
It's a mistake. I originally depended on this target directly but I decided not to do this anymore because it adds at least a few secs to the build even if there is nothing to install. Will revert this change.